ravedude/src/main.rs

```diff
@@ -257,6 +264,12 @@ fn ravedude() -> anyhow::Result<()> {
         anyhow::bail!("no named board given and no board options provided");
     };
 
+    if ravedude_config.general_options.newline_on.is_some()
+        && ravedude_config.general_options.newline_after.is_some()
+    {
+        anyhow::bail!("newline_on and newline_after cannot be used at the same time");
+    }
+
```

Comment:
This check can now be dropped that we ensure the same thing in `RavedudeGeneralConfig::newline_mode()`, right?

Reply:
true, good catch!